integration-tests/md/extend.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# requirements: md raid /dev/md0 with partitions sdb[1-4] and unused partition sdb5
+
+
+from sys import exit
+from storage import *
+from storageitu import *
+
+
+set_logger(get_logfile_logger())
+
+environment = Environment(False)
+
+storage = Storage(environment)
+
+staging = storage.get_staging()
```

Comment:
`get_staging_devicegraph()` would be more readable for me.  In C++ code it would be probably obvious, with explicitly declared types.
https://github.com/openSUSE/libstorage-ng/blob/dd5d8cfa5220b570e188f4e18621287ee15c5c10/storage/Storage.h#L113